app/src/main/java/luci/sixsixsix/powerampache2/player/SimpleMediaServiceHandler.kt

```diff
@@ -217,6 +210,7 @@ class SimpleMediaServiceHandler @Inject constructor(
             ExoPlayer.STATE_IDLE -> {
                 L("STATE_IDLE")
                 _simpleMediaState.value = SimpleMediaState.Idle
+                retryPlay()
```

Comment:
double check this, player might go on error, that's a more appropriate place to call retry